webapp/channels/src/components/app.tsx

```diff
@@ -16,20 +16,18 @@ const LazyRoot = React.lazy(() => import('components/root'));
 
 const Root = makeAsyncComponent('Root', LazyRoot);
 
-class App extends React.PureComponent {
-    render() {
-        return (
-            <Provider store={store}>
-                <CRTPostsChannelResetWatcher/>
-                <Router history={getHistory()}>
-                    <Route
-                        path='/'
-                        component={Root}
-                    />
-                </Router>
-            </Provider>
-        );
-    }
+function App() {
```

Comment:
Just for consistency with other parts of the code, we prefer using arrow functions.
```suggestion
const App = () => {
```